common/src/main/java/com/flipkart/varadhi/common/Constants.java

```diff
@@ -52,6 +46,10 @@ private PathParams() {
 
     public static class ContextKeys {
         public static final String USER_CONTEXT = "userContext";
+        public static final String API_RESPONSE = "api-response";
```

Comment:
why hyphen here?

Reply:
No reason. I simply moved the variables around. values got ignored. will convert to camelcase